DEBUG    twister_harness.device.factory:factory.py:37 Get device type "native"
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d172cce10>
DEBUG    httpcore.connection:_trace.py:87 start_tls.started ssl_context=<ssl.SSLContext object at 0x7f1d1722fd10> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:87 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d1713fe10>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:87 send_request_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Access-Control-Expose-Headers', b'Content-Range'), (b'Content-Length', b'188'), (b'Content-Range', b'projects 0-1/1'), (b'Content-Type', b'application/json'), (b'Date', b'Wed, 27 Nov 2024 15:40:19 GMT'), (b'Grpc-Metadata-Access-Control-Expose-Headers', b'Content-Range'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'0'), (b'X-Kong-Upstream-Latency', b'15')])
INFO     httpx:_client.py:1786 HTTP Request: GET https://api.golioth.io/v1/projects "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:87 response_closed.started
DEBUG    httpcore.http11:_trace.py:87 response_closed.complete
DEBUG    httpcore.connection:_trace.py:87 close.started
DEBUG    httpcore.connection:_trace.py:87 close.complete
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d172c0d90>
DEBUG    httpcore.connection:_trace.py:87 start_tls.started ssl_context=<ssl.SSLContext object at 0x7f1d170f49e0> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:87 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d17290f50>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:87 send_request_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:87 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'287'), (b'Content-Type', b'application/json'), (b'Date', b'Wed, 27 Nov 2024 15:40:19 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'1'), (b'X-Kong-Upstream-Latency', b'106')])
INFO     httpx:_client.py:1786 HTTP Request: POST https://api.golioth.io/v1/projects/firmware_ci/devices "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:87 response_closed.started
DEBUG    httpcore.http11:_trace.py:87 response_closed.complete
DEBUG    httpcore.connection:_trace.py:87 close.started
DEBUG    httpcore.connection:_trace.py:87 close.complete
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d170f18d0>
DEBUG    httpcore.connection:_trace.py:87 start_tls.started ssl_context=<ssl.SSLContext object at 0x7f1d170f5130> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:87 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d1717d110>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:87 send_request_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:87 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'201'), (b'Content-Type', b'application/json'), (b'Date', b'Wed, 27 Nov 2024 15:40:19 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'1'), (b'X-Kong-Upstream-Latency', b'31')])
INFO     httpx:_client.py:1786 HTTP Request: POST https://api.golioth.io/v1/projects/firmware_ci/devices/67473d63bb6157cf6cdfbb5a/credentials "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.started request=<Request [b'POST']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:87 response_closed.started
DEBUG    httpcore.http11:_trace.py:87 response_closed.complete
DEBUG    httpcore.connection:_trace.py:87 close.started
DEBUG    httpcore.connection:_trace.py:87 close.complete
DEBUG    twister_harness.device.binary_adapter:utils.py:37 Running command: /__w/golioth-firmware-sdk/golioth-firmware-sdk/twister-out/native_sim_native_64/logging/sample.golioth.logging/zephyr/zephyr.exe
DEBUG    twister_harness.helpers.utils:utils.py:22 Found matching key: CONFIG_SHELL_PROMPT_UART="uart:~$ "
INFO     twister_harness.fixtures:fixtures.py:75 Wait for prompt
DEBUG    twister_harness.helpers.shell:shell.py:47 Got prompt
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d171b1cd0>
DEBUG    httpcore.connection:_trace.py:87 start_tls.started ssl_context=<ssl.SSLContext object at 0x7f1d1722f260> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:87 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d1717fe50>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:87 send_request_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Access-Control-Expose-Headers', b'Content-Range'), (b'Content-Length', b'203'), (b'Content-Range', b'credentials 0-1/1'), (b'Content-Type', b'application/json'), (b'Date', b'Wed, 27 Nov 2024 15:40:21 GMT'), (b'Grpc-Metadata-Access-Control-Expose-Headers', b'Content-Range'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'1'), (b'X-Kong-Upstream-Latency', b'29')])
INFO     httpx:_client.py:1786 HTTP Request: GET https://api.golioth.io/v1/projects/firmware_ci/devices/67473d63bb6157cf6cdfbb5a/credentials "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:87 response_closed.started
DEBUG    httpcore.http11:_trace.py:87 response_closed.complete
DEBUG    httpcore.connection:_trace.py:87 close.started
DEBUG    httpcore.connection:_trace.py:87 close.complete
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [msettings set golioth/psk-id generated-yKQIBDwaBEMjRmIR@firmware_ci
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting golioth/psk-id to generated-yKQIBDwaBEMjRmIR@firmware_ci
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting golioth/psk-id saved as generated-yKQIBDwaBEMjRmIR@firmware_ci
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [msettings set golioth/psk generated-yKQIBDwaBEMjRmIR
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting golioth/psk to generated-yKQIBDwaBEMjRmIR
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: Setting golioth/psk saved as generated-yKQIBDwaBEMjRmIR
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:05.130,003] [0m<inf> golioth_coap_client_zephyr: Golioth CoAP client connected[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:05.130,003] [0m<dbg> golioth_logging: main: Debug info! 0[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:05.130,003] [0m<dbg> golioth_logging: func_1: Log 1: 0[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:05.130,003] [0m<dbg> golioth_logging: func_2: Log 2: 0[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:05.130,003] [1;33m<wrn> golioth_logging: Warn: 0[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:05.130,003] [1;31m<err> golioth_logging: Err: 0[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:05.130,003] [0m<inf> golioth_logging: Counter hexdump
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: 00 00 00 00                                      |....             [0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:05.130,003] [0m<inf> golioth_logging: Golioth client connected[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:05.130,003] [0m<inf> golioth_coap_client_zephyr: Entering CoAP I/O loop[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[1;32muart:~$ [m[8D[J[00:00:10.140,000] [0m<dbg> golioth_logging: main: Debug info! 1[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:10.140,000] [0m<dbg> golioth_logging: func_1: Log 1: 1[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:10.140,000] [0m<dbg> golioth_logging: func_2: Log 2: 1[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:10.140,000] [1;33m<wrn> golioth_logging: Warn: 1[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:10.140,000] [1;31m<err> golioth_logging: Err: 1[0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[00:00:10.140,000] [0m<inf> golioth_logging: Counter hexdump
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: 00 00 00 01                                      |....             [0m
DEBUG    twister_harness.device.device_adapter:device_adapter.py:148 #: [1;32muart:~$ [m[8D[J[1;32muart:~$ [m[8D[J[00:00:15.150,000] [0m<dbg> golioth_logging: main: Debug info! 2[0m
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d171b1690>
DEBUG    httpcore.connection:_trace.py:87 start_tls.started ssl_context=<ssl.SSLContext object at 0x7f1d170f4d40> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:87 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d171b1c50>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:87 send_request_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Access-Control-Expose-Headers', b'Content-Range'), (b'Content-Range', b'logs 0-15/15'), (b'Content-Type', b'application/json'), (b'Date', b'Wed, 27 Nov 2024 15:40:36 GMT'), (b'Grpc-Metadata-Access-Control-Expose-Headers', b'Content-Range'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'0'), (b'X-Kong-Upstream-Latency', b'76'), (b'Transfer-Encoding', b'chunked')])
INFO     httpx:_client.py:1786 HTTP Request: GET https://api.golioth.io/v1/projects/firmware_ci/logs?start=2024-11-27T15%3A40%3A21.906943Z&end=2024-11-27T15%3A40%3A35.728472Z&deviceId=67473d63bb6157cf6cdfbb5a "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.started request=<Request [b'GET']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:87 response_closed.started
DEBUG    httpcore.http11:_trace.py:87 response_closed.complete
DEBUG    httpcore.connection:_trace.py:87 close.started
DEBUG    httpcore.connection:_trace.py:87 close.complete
INFO     test_sample:test_sample.py:26 Searching log messages:
INFO     test_sample:test_sample.py:37 Golioth CoAP client connected
INFO     test_sample:test_sample.py:37 ### MATCH FOUND! ---> main: Debug info! 0
INFO     test_sample:test_sample.py:37 ### MATCH FOUND! ---> func_1: Log 1: 0
INFO     test_sample:test_sample.py:37 ### MATCH FOUND! ---> func_2: Log 2: 0
INFO     test_sample:test_sample.py:37 ### MATCH FOUND! ---> Warn: 0
INFO     test_sample:test_sample.py:37 ### MATCH FOUND! ---> Err: 0
INFO     test_sample:test_sample.py:37 ### MATCH FOUND! ---> Counter hexdump
INFO     test_sample:test_sample.py:48 ### All expected Log messages found!
DEBUG    twister_harness.device.binary_adapter:binary_adapter.py:87 Running subprocess finished with return code 0
DEBUG    httpx:_config.py:82 load_ssl_context verify=True cert=None trust_env=True http2=False
DEBUG    httpx:_config.py:148 load_verify_locations cafile='/usr/local/lib/python3.11/dist-packages/certifi/cacert.pem'
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.started host='api.golioth.io' port=443 local_address=None timeout=5.0 socket_options=None
DEBUG    httpcore.connection:_trace.py:87 connect_tcp.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d171b2c50>
DEBUG    httpcore.connection:_trace.py:87 start_tls.started ssl_context=<ssl.SSLContext object at 0x7f1d1722f1d0> server_hostname='api.golioth.io' timeout=5.0
DEBUG    httpcore.connection:_trace.py:87 start_tls.complete return_value=<httpcore._backends.trio.TrioStream object at 0x7f1d171b3190>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.started request=<Request [b'DELETE']>
DEBUG    httpcore.http11:_trace.py:87 send_request_headers.complete
DEBUG    httpcore.http11:_trace.py:87 send_request_body.started request=<Request [b'DELETE']>
DEBUG    httpcore.http11:_trace.py:87 send_request_body.complete
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.started request=<Request [b'DELETE']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Content-Length', b'2'), (b'Content-Type', b'application/json'), (b'Date', b'Wed, 27 Nov 2024 15:40:36 GMT'), (b'Grpc-Metadata-Content-Type', b'application/grpc'), (b'Via', b'kong/2.5.2'), (b'X-Kong-Proxy-Latency', b'1'), (b'X-Kong-Upstream-Latency', b'19')])
INFO     httpx:_client.py:1786 HTTP Request: DELETE https://api.golioth.io/v1/projects/firmware_ci/devices/67473d63bb6157cf6cdfbb5a "HTTP/1.1 200 OK"
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.started request=<Request [b'DELETE']>
DEBUG    httpcore.http11:_trace.py:87 receive_response_body.complete
DEBUG    httpcore.http11:_trace.py:87 response_closed.started
DEBUG    httpcore.http11:_trace.py:87 response_closed.complete
DEBUG    httpcore.connection:_trace.py:87 close.started
DEBUG    httpcore.connection:_trace.py:87 close.complete